hydromt/model/components/vector.py

```diff
@@ -341,15 +341,19 @@ def write(
             )
         # write to geojson only
         elif filename is None:
-            os.makedirs(dirname(join(self.root.path, geometry_filename)), exist_ok=True)
+            full_path = join(self.root.path, geometry_filename)
+            os.makedirs(dirname(full_path), exist_ok=True)
             gdf = ds.vector.to_gdf(**kwargs)
-            gdf.to_file(join(self.root.path, geometry_filename))
+            logger.debug(f"Writing file {full_path}")
```

Comment:
In this particular case, I would also write a clearer message. The first one above should be "Writting {component_name} data and geometry to netcdf file {full_path}".
The current one: "Writting {component_name} data and geometry to geojson file {full_path}".
THe one below: "Writting {component_name} data to netcdf file {filepath} and geometry to geojson file {full_path}"